projects/igniteui-angular/src/lib/directives/mask/mask-parsing.service.ts

```diff
@@ -0,0 +1,251 @@
+import { Injectable } from '@angular/core';
```

Comment:
Huh, apparently really hard for github to display the rename +diff...

Reply:
Nvm, `git log --follow` shows the old file history just fine, it's just `git diff` that can't handle the rename with that much changes apparently :/

Reply:
Sigh.
`rename projects/igniteui-angular/src/lib/directives/mask/{mask-helper.ts => mask-parsing.service.ts} (42%)`. We went over the threshold there, Skippy.
@jackofdiamond5 Can you do the file rename in a separate PR (will merge first) so this can have  a decent diff?